src/main/java/com/betting/ground/admin/repository/ReportRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.betting.ground.admin.repository;
+
+import com.betting.ground.admin.domain.Report;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface ReportRepository extends JpaRepository<Report, Long> {
+    List<Report> searchReportById(Long id);
```

Comment:
여기서 의미하는 id는 report의 id인가요? 그렇다면
```suggestion
Optional<Report> searchReportById(Long id);
```
로 받으면 어떨까요?
userId나 auctionId로 찾는 거라면 메서드 명을 좀 더 명확하게 하는 게 어떨까요

Reply:
한 유저가 신고를 여러번 할수있기때문에
List로 그대로 두었습니다

Reply:
이해했습니다!

Reply:
수정했습니다